tests/rptest/services/redpanda.py

```diff
@@ -505,3 +521,7 @@ def delete_topic(self, name):
         client = self._client_type(self)
         self.logger.debug(f"Deleting topic {name}")
         client.delete_topic(name)
+
+    def cov_enabled(self):
+        return self._context.globals.get(self.COV_KEY,
+                                         self.DEFAULT_COV_OPT) == "yes"
```

Comment:
I think the global are parsed as json objects, so you should be able to pass in normal true/false types like:

```diff
nwatkins@gordon:~/src/redpanda/vtools$ git diff
diff --git a/taskfiles/rp.yml b/taskfiles/rp.yml
index 2e6d555..171adb6 100644
--- a/taskfiles/rp.yml
+++ b/taskfiles/rp.yml
@@ -172,7 +172,7 @@ tasks:
           --cluster-file=/cluster.json
           --results-root=/build/tests/results
           --max-parallel={{.DUCKTAPE_MAX_PARALLEL}}
-          --globals='{"rp_install_path_root":"{{.RP_INSTALL_DIR}}","redpanda_log_level":"{{.REDPANDA_LOG_LEVEL}}","s3_bucket":"{{.S3_BUCKET}}","s3_region":"{{.S3_REGION}}","s3_access_key":"{{.S3_ACCESS_KEY}}","s3_secret_key":"{{.S3_SECRET_KEY}}","scale":"{{.DUCKTAPE_SCALE}}"}'
+          --globals='{"enable_cov":true,"rp_install_path_root":"{{.RP_INSTALL_DIR}}","redpanda_log_level":"{{.REDPANDA_LOG_LEVEL}}","s3_bucket":"{{.S3_BUCKET}}","s3_region":"{{.S3_REGION}}","s3_access_key":"{{.S3_ACCESS_KEY}}","s3_secret_key":"{{.S3_SECRET_KEY}}","scale":"{{.DUCKTAPE_SCALE}}"}'
           {{.DUCKTAPE_ARGS}}
 
   stop-compose-cluster:
```

Reply:
I have that on my PR on vtools side. See https://github.com/vectorizedio/vtools/pull/246/files
I just have some strange CI failure to work though there though.